scripts/setupLocalIdler.sh

```diff
@@ -226,14 +230,41 @@ env() {
     login
     setTokens
 
-    echo export JC_OPENSHIFT_API_URL=https://api.free-stg.openshift.com
-    echo export JC_OPENSHIFT_API_TOKEN=${JC_OPENSHIFT_API_TOKEN}
-    echo export JC_AUTH_TOKEN=${JC_AUTH_TOKEN}
     echo export JC_IDLE_AFTER=${LOCAL_DEFAULT_IDLE_TIME}
     echo export JC_JENKINS_PROXY_API_URL=http://localhost:${LOCAL_PROXY_PORT}
     echo export JC_F8TENANT_API_URL=http://localhost:${LOCAL_TENANT_PORT}
     echo export JC_TOGGLE_API_URL=http://localhost:${LOCAL_TOGGLE_PORT}/api
+    echo export JC_AUTH_URL=https://auth.prod-preview.openshift.io
+    echo export JC_SERVICE_ACCOUNT_ID=${JC_SERVICE_ACCOUNT_ID}
+    echo export JC_SERVICE_ACCOUNT_SECRET=\"${JC_SERVICE_ACCOUNT_SECRET}\"
+    echo export JC_AUTH_TOKEN_KEY=\"${JC_AUTH_TOKEN_KEY}\"
     echo export JC_FIXED_UUIDS=${JC_FIXED_UUIDS}
+    echo export JC_OPENSHIFT_API_URL=https://api.free-stg.openshift.com
+    echo export JC_OPENSHIFT_API_TOKEN=\"${JC_OPENSHIFT_API_TOKEN}\"
+    echo export JC_AUTH_TOKEN=${JC_AUTH_TOKEN}
```

Comment:
Should all tokens get quoted (like you have done previously) in case there is a dollar bracket or other things that the shell would not like when doing the eval,

Reply:
but this is quoted now. Also JC_OPENSHIFT_API_TOKEN will go shortly

Reply:
okay sure,